README.md

```diff
@@ -74,5 +74,22 @@ prob_func = (prob, i, repeat) -> remake(prob, p = (@SVector rand(Float32, 3)) .*
 monteprob = EnsembleProblem(prob, prob_func = prob_func, safetycopy = false)
 
 @time sol = solve(monteprob, GPUTsit5(), EnsembleGPUKernel(CUDA.CUDABackend()),
-                  trajectories = 10_000, adaptive = false, dt = 0.1f0)
+    trajectories = 10_000, adaptive = false, dt = 0.1f0)
+```
+
+## Benchmarks
+
+Curious about our claims? See [https://github.com/utkarsh530/GPUODEBenchmarks](https://github.com/utkarsh530/GPUODEBenchmarks) for comparsion of our GPU solvers against CPUs and GPUs implementation in C++, JAX and PyTorch.
+
```

Comment:
We should try to get this setup on the SciMLBenchmarks servers. @staticfloat @maleadt could we get help getting a GPU queue for benchmarking?

Reply:
There already is a `benchmark` queue that's used for CUDA.jl. Contains a single runner, albeit a fairly underpowered one.

Reply:
We'd need some way to select which compute resource to do in each benchmark folder.